test-app/ember-cli-build.js

```diff
@@ -17,6 +17,11 @@ module.exports = function (defaults) {
 
   const options = {
     'ember-bootstrap': {},
+    babel: {
+      plugins: [
+        require.resolve('ember-concurrency/async-arrow-task-transform'),
+      ],
+    },
```

Comment:
Can we please remove this? The test app does not use `ember-concurrency` directly at all.

```suggestion
```

I think we can even remove the `devDependency` on `ember-concurrency` in the test app. If we want to ensure a specific version being used to run the tests against, we could use [`pnpm.overrides`](https://pnpm.io/package_json#pnpmoverrides) instead. But as that is not only true for `ember-concurrency` but for many other dependencies as well, let's discuss that in a separate PR.

Reply:
no problem. I removed it from the test app